src/Microsoft.Azure.WebJobs.Host/JobHostConfiguration.cs

```diff
@@ -123,6 +125,31 @@ public string HostId
             }
         }
 
+        /// <summary>
+        /// Gets the host machine ID. All host instances on the same machine will have the same HostMachineId.
```

Comment:
It's certainly possible to have two machines with the same name (although is that possible in Antares?) -- but highly unlikely. If this happens, the Lease will keep getting exchanged back-and-forth between those machines. We'd have the same issue in the ScriptHost if this happens -- two hosts would think they're the primary host.

@fabiocav -- when doing this from ScriptHost, did you give thought to anything else on the machine that would work better than the machine name?